.travis.yml

```diff
@@ -161,14 +185,13 @@ install:
     - test "$TRAVIS_OS_NAME" != "linux" || (scripts/install_cmake.sh)
     # Disable tests unless run on the release branch, on tags or with daily cron
     #- if [ "$TRAVIS_BRANCH" != release -a -z "$TRAVIS_TAG" -a "$TRAVIS_EVENT_TYPE" != cron ]; then SOLC_TESTS=Off; fi
-    - SOLC_TESTS=Off
-    - if [ "$TRAVIS_BRANCH" = release -o -n "$TRAVIS_TAG" ]; then echo -n > prerelease.txt; else date -u +"nightly.%Y.%-m.%-d" > prerelease.txt; fi
-    - echo -n "$TRAVIS_COMMIT" > commit_hash.txt
 
 before_script:
+    - if [ "$TRAVIS_BRANCH" = release -o -n "$TRAVIS_TAG" ]; then echo -n > prerelease.txt; else date -u +"nightly.%Y.%-m.%-d" > prerelease.txt; fi
+    - echo -n "$TRAVIS_COMMIT" > commit_hash.txt
     - test $SOLC_EMSCRIPTEN != On || (scripts/build_emscripten.sh)
     - test $SOLC_DOCKER != On || (scripts/docker_build.sh)
-    - test $SOLC_RELEASE != On || (scripts/build.sh $SOLC_BUILD_TYPE
+    - test $SOLC_RELEASE != On || (scripts/build.sh $SOLC_BUILD_TYPE -DBoost_USE_STATIC_LIBS=OFF
```

Comment:
Can you explain this change? Will this need the boost so's in the generated zip now?

Reply:
yes, please remove it.

Reply:
Done.

Reply:
I added the build alone back, though, since we'll probably still want to have test runs.